content/docs/how-we-work/ci-cd.md

```diff
@@ -14,6 +14,17 @@ Like most errors we see when dealing with computers, errors reported by CI are n
 
 GitHub Actions tout themselves as an automation platform, not just CI/CD. In our projects, these are mostly used for CI/CD tasks. It is useful to know how you could run other forms of automation to manage a project on GitHub but we won't focus on that here.
 
+To set up GitHub Actions for your project, follow these steps:
+
+- In the branch, you would like to work (usually a feature branch out of the `main` branch), create a folder named `.github` in the root of your project directory.
+- Create a folder named `workflows` in the `.github` directory. This is where you'll create all your YAML files.
+- Create a YAML file (e.g., ci.yml) in the workflows directory and add your workflow configuration.
```

Comment:
This is too long. It can be condensed to:

> Create `.github/workflows` directory in your project root. You can add your workflow files here. Refer to our templates and GitHub Actions Manual for more details on what the workflow files look like.

Reply:
I have consciously left out the bit about branches. I don't think it's necessary to spell it out because this may create confusion. For example, they may add the files through a PR and that means that it won't be in the default branch in the beginning.

If it is important to say, then add on a short text like this:

> For the workflow files to be picked up by GitHub, they need to be in the default branch (e.g., `main`).